data/hai/hai reveal 1 intro.txt

```diff
@@ -36,7 +36,6 @@ mission "Hai Reveal: Reputation Reset"
 		"reputation: Republic that won't enter wormhole" = "reputation: Republic"
 		"reputation: Free Worlds that won't enter wormhole" = "reputation: Free Worlds"
 		# Make sure the new governments start with appropriate rep:
-		"reputation: Hai (Friendly Unfettered)" = 1000
 		"reputation: Elenctic Commune" = 1000
```

Comment:
Why does this government have such a positive attitude towards the Hai and Friendly Unfettered (0.1) and such negative attitudes towards Pirates, the Devil-Run Gang, and the Smugglers (-0.2) if you want it to start at 1000 anyway?

Reply:
Presumably because it was a convenient safety factor when the government was created.

Reply:
Do you mean the same incorrect assumption about how player reputations in government definitions work as with the other government here or something else?

Reply:
I mean, your question is about values that are not present at the line you're commenting on - so that's what was being addressed - so no.
I believe it is set like that though because _before_ the story starts I believe the government will still have opinions change based on your actions, hence the intentional reset that doesn't entirely negate player agency about what they want to do before encountering this group.
But as for the ratio of opinions that you initially appear to have asked about; that is a legacy of nomadic's

Reply:
I brought up those values because they affect the way this government behaves. And that behaviour is highly relevant to this line.

> I believe it is set like that though because before the story starts I believe the government will still have opinions change based on your actions, hence the intentional reset that doesn't entirely negate player agency about what they want to do before encountering this group.

If the magnitudes of those opinions are important (that is, you want it to be possible to increase reputation with them), I suggest adding those by events instead of resetting the reputation at somewhat arbitrary locations (and, if you must reset the reputation, you should do it closer to when the government is actually first used.)

> But as for the ratio of opinions that you initially appear to have asked about; that is a legacy of nomadic's

So, are they important or not?